modules/stats.php

```diff
@@ -36,6 +36,16 @@
 	'tag_base'
 );
 
+function is_dnt_enabled() {
+	foreach ($_SERVER as $name => $value) {
+		if ( strtolower( $name ) == 'http_dnt' && $value == 1 ) {
```

Comment:
Could you use [Yoda conditions](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions) here?